### PR TITLE
fix: video and placehoder image overlapping

### PR DESCRIPTION
**Pull request by alantoa**
# Why

- video and image overlapping.
- some components have type error/warning
- we removed the Collection component, so we need to adjust item height.
- the style of the card is inconsistent with Figma.
- mint modal style has some broken.
# How

- blur the background image and center the video, and add a patch for `expo-av`.
- fix many types of errors.
- adjust card item height.
- improve card style.
- fix mint modal style.
# Test Plan

- check display normally if the video and image overlapping.
- check if the card style is looking good.
- check if the mint modal is looking good.

### CODE REVIEW

File: packages/design-system/spinner/index.web.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import { useEffect, useRef, useCallback, useMemo } from "react";
+import { useEffect, useCallback, useMemo } from "react";
 import { Animated, Easing } from "react-native";
 
 import { getSpinnerSize, SpinnerView, SpinnerProps } from "./spinner-view";
 
 export const Spinner = ({ size, duration = 750, ...rest }: SpinnerProps) => {
-  const transition = useRef(new Animated.Value(0)).current;
-
+  const transition = useMemo(() => new Animated.Value(0), []);
```
Comment (catalinmiron) [inline packages/design-system/spinner/index.web.tsx]: It's better to use `useRef` for Animated.Values. It's safe to revert this change. 
Also, what do you think about using `moti` instead? 

```javascript
<MotiView 
  from={{rotate: '0deg'}}
  animate={{rotate: '360deg'}}
  transition={{
    type: 'timing',
    loop: true,
    repeatReverse: false,
    easing: Easing.linear
  }}
  style={{
    height: getSpinnerSize(size),
    width: getSpinnerSize(size),
  }}
/>
```

File: packages/app/hooks/use-nft-cards-list-layout-provider.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ const useNFTCardDimensions = () => {
     return { height: width / 3, width: width / 3 };
   }
   return {
-    height: 594 + LIST_MARGIN_TOP_LARGE,
+    height: 548 + LIST_MARGIN_TOP_LARGE,
```
Comment (catalinmiron) [inline packages/app/hooks/use-nft-cards-list-layout-provider.ts]: can we move this to a constant?

File: packages/app/components/feed/comment-button.tsx [hunk 3]
```diff
@@ -50,8 +52,9 @@ export function CommentButton({ nft }: CommentButtonProps) {
       size="regular"
       tw="h-auto p-0"
       onPress={handleOnPress}
+      accentColor={textColors}
     >
-      <Message height={24} width={24} />{" "}
+      <Message height={24} width={24} color={iconColor} />{" "}
       {nft?.comment_count > 0 ? getRoundedCount(nft.comment_count) : ""}
```
Comment (catalinmiron) [inline packages/app/components/feed/comment-button.tsx:58]: I am not sure if this is semantically correct, should we wrap the "content" into a `<Text>` element? This should work on web, but React Native likes to keep the "text" wrapped into a <Text> element.

File: packages/app/components/claim/gift-button.tsx [hunk 4]
```diff
@@ -1,26 +1,28 @@
...
-        // @ts-ignore
-        color={tw.style("bg-gray-900 dark:bg-white").backgroundColor}
-      />{" "}
+    <Button
+      variant="text"
+      size="regular"
+      tw="h-auto p-0"
+      accentColor={textColors}
+    >
+      <Gift height={20} width={20} color={iconColor} />{" "}
```
Comment (catalinmiron) [inline packages/app/components/claim/gift-button.tsx:25]: Same

**Review by catalinmiron — CHANGES_REQUESTED** [review]

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM 💯

File: packages/app/components/card/rows/elements/ownership.tsx [hunk 5]
```diff
@@ -26,7 +26,7 @@ function OwnershipContainer({
   ) : (
     <View
       tw={[
-        "flex h-[30px] w-[30px] flex-row flex-wrap justify-between",
+        "flex h-[32px] w-[32px] flex-row flex-wrap justify-between",
```
Comment (intergalacticspacehighway) [inline packages/app/components/card/rows/elements/ownership.tsx]: nit: here i think we can use h-8, w-8

Comment (alantoa) [reply packages/app/components/card/rows/elements/ownership.tsx]: yeah agree!

File: packages/design-system/spinner/index.web.tsx (hunk 1 above)
Comment (alantoa) [reply packages/design-system/spinner/index.web.tsx]: thank you for your review!
it's just a test, I'm not sure if it's causing every update [this](https://www.chromatic.com/build?appId=619f8881c3b4c2003a08edf7&number=2092), `moti` I'll try again, I can't remember what problem I had before without using it.

File: packages/app/hooks/use-nft-cards-list-layout-provider.ts (hunk 2 above)
Comment (alantoa) [reply packages/app/hooks/use-nft-cards-list-layout-provider.ts]: yeah that's awosome!

File: packages/app/components/feed/comment-button.tsx (hunk 3 above)
Comment (alantoa) [reply packages/app/components/feed/comment-button.tsx:58]: yeah, it's works on native/web, I very much agree with your suggestion!  I also like the `${text} ` or `&nbsp;` way of writing, I will modify it.
It's just that it was written like this before, I didn't change it, maybe we define a specification.

File: packages/app/components/feed/feed.md.tsx [hunk 6]
```diff
@@ -30,7 +30,7 @@ import { Hidden } from "design-system/hidden";
 import { Tabs } from "design-system/tabs";
 import { CARD_DARK_SHADOW } from "design-system/theme";
 
-const CARD_HEIGHT = 848;
+const CARD_HEIGHT = 825;
```
Comment (enesozturk) [inline packages/app/components/feed/feed.md.tsx:33]: Any reason to change this? We sometimes needed to change these values but they are causing UI bugs. I think we should use normal the web approach instead rendering the lists with FlatList or VirtualizedList etc. I'll open a discussion on this. Just wanted to ping you if this won't break something.

**Review by catalinmiron — APPROVED** [review]

**Comment by catalinmiron** [comment]
LGTM

**Review by enesozturk — APPROVED** [review]
LGTM ❇️

File: packages/app/components/feed/feed.md.tsx (hunk 6 above)
Comment (alantoa) [reply packages/app/components/feed/feed.md.tsx:33]: yeah, we have removed the Collection component, so we need to adjust item height.
I know now fix height it's very hacking, It will be resolved once we get rid of the `recyclerlistview`.

File: packages/design-system/spinner/index.web.tsx (hunk 1 above)
Comment (alantoa) [reply packages/design-system/spinner/index.web.tsx]: just try it:
image
it's working on the web but does not work on native. :(

Comment (alantoa) [reply packages/design-system/spinner/index.web.tsx]: but thank you tips!
so let's try using `moti` on the web, native code is temporarily unchanged, see [this](https://www.chromatic.com/build?appId=619f8881c3b4c2003a08edf7&number=2092) if it disappears
